### PR TITLE
pageobject and properties file features completed

### PR DESCRIPTION
**Pull request by Answeroid**
A lot of changes made about pageobject pattern (at least how I see it)
config file with properties also done there

please review and I have a couple of questions to ask after.

thanks

### CODE REVIEW

File: src/main/java/HomePage.java [hunk 1]
```diff
@@ -1,35 +1,71 @@
...
 
-    public LoginPage logOutFromParkMe() {
-        driver.findElement(By.xpath(".//*[@id='header_fullsize']/div[1]/span[2]/div/div/div[1]/span")).click();
-        driver.findElement(By.xpath(".//*[@id='header_fullsize']/div[1]/span[2]/div/div/ul/li[5]/a")).click();
+    public void login_negative() {
+        loginPageButtonLink.click();
+        usernameField.sendKeys(Config.getSetting("incorrectUsername"));
+        passwordField.sendKeys(Config.getSetting("incorrectPassword"));
+        loginButton.click();
+    }
```
Comment (boshasiv) [inline src/main/java/HomePage.java]: should be one login method which accepts two parameters: usrname and pwd, so tester can define what exactly he wants to test

**Review by boshasiv — CHANGES_REQUESTED** [review]
in general, in xpathes there are shouldn't be any numbers like div[1], just try to get rid of it

File: src/main/java/HomePage.java (hunk 1 above)
Comment (vkhalin) [reply src/main/java/HomePage.java]: fixed on new commits

File: src/main/java/SearchResultPage.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+
+public class SearchResultPage extends Page {
+
+    @FindBy(xpath = "html/body/div[2]/div[2]/div[2]/div[2]/div[1]")
```
Comment (boshasiv) [inline src/main/java/SearchResultPage.java]: what about this one?

**Review by boshasiv — APPROVED** [review]

**Pull request merged**